src/index.html

```diff
@@ -10,7 +10,57 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    data-qa="big"
+    class="form"
+    >
+      <label
+        for="big__search"
```

Comment:
The `for `attribute of the label should be rather written without an underscore. Change the 'for' attribute to `big-search` or `bigSearch`.